library/src/main/java/com/novoda/downloadmanager/lib/DownloadsRepository.java

```diff
@@ -59,6 +63,37 @@ public FileDownloadInfo getDownloadFor(long id) {
         return downloadInfoCreator.create(reader, id);
     }
 
+    public void moveDownloadsStatusTo(List<Long> ids, int status) {
+        if (ids.isEmpty()) {
+            return;
+        }
+
+        ContentValues values = new ContentValues(1);
+        values.put(DownloadContract.Downloads.COLUMN_STATUS, status);
+
+        String where = DownloadContract.Downloads._ID + " IN (" + getPlaceHoldersOf(ids.size()) + ")";
+        String[] selectionArgs = getStringArrayFrom(ids.toArray());
+        contentResolver.update(downloadsUriProvider.getAllDownloadsUri(), values, where, selectionArgs);
+    }
+
+    private String getPlaceHoldersOf(int size) {
```

Comment:
Notils upgrade was created because of this and decided to modify it in another PR in order to keep this small. What @danybony said :)